### PR TITLE
Remove Add CANS link when disabled

### PR DESCRIPTION
**Pull request by ggomez-cambriasolutions**
## Description
If the user does not have permission to add a cans for a client, then we shouldn't render the "Add CANS" link on the Client Assessment History. If they do, then the "Add CANS" link will be visible and clickable. 

## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by ggomez-cambriasolutions** [comment]
jenkins retest this please

File: app/javascript/Application/components/Client/AssessmentHistory/ClientAssessmentHistory.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ class ClientAssessmentHistory extends PureComponent {
     const { clientIdentifier } = this.props
```
Comment (pokharelmanish) [inline app/javascript/Application/components/Client/AssessmentHistory/ClientAssessmentHistory.js]: Is this use anywhere now?

**Comment by pokharelmanish** [comment]
retest this please

**Comment by pokharelmanish** [comment]
retest this please

File: app/javascript/Application/components/Client/AssessmentHistory/ClientAssessmentHistory.js (hunk 1 above)
Comment (ggomez-cambriasolutions) [reply app/javascript/Application/components/Client/AssessmentHistory/ClientAssessmentHistory.js]: Yes, it is used by `buildCreateAssessmentPermission(clientIdentifier)` to build the permission that we check in the AuthBoundary.

Comment (pokharelmanish) [reply app/javascript/Application/components/Client/AssessmentHistory/ClientAssessmentHistory.js]: oh my bad. I didn't notice that.

**Comment by ggomez-cambriasolutions** [comment]
jenkins retest this please

**Comment by ggomez-cambriasolutions** [comment]
jenkins retest this please

**Comment by ggomez-cambriasolutions** [comment]
jenkins retest this please

**Pull request merged**